buildSrc/src/main/kotlin/com/toasttab/protokt/gradle/ProtobufBuild.kt

```diff
@@ -73,6 +75,14 @@ private fun extraClasspath(project: Project, task: GenerateProtoTask): String {
         extensions += project.configurations.getByName(TEST_EXTENSIONS)
     }
 
+    // Must explicitly register input files here; if any extensions dependencies are project dependencies then Gradle
+    // won't pick them up as dependencies unless we do this. There may be a better way to do this but for now just
+    // manually do what protobuf-gradle-plugin used to do.
+    // https://github.com/google/protobuf-gradle-plugin/commit/0521fe707ccedee7a0b4ce0fb88409eefb04e59d
+    project.tasks.withType<ProtobufExtract>()
+        .filter { it.name.startsWith("extractInclude") }
```

Comment:
hm does this configure the task eagerly? should this be `tasks.withType<> { if (name.startsWith) ... }` instead?

Reply:
Done